roles/byzantine/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+# tasks file for roles/special_collections
+
+# needed for capistrano to have access to this file
+- name: byzantine | Check to see if .env.local file exists.
+  ansible.builtin.stat:
+    path: "/home/{{ deploy_user }}/.env.local"
+  register: env_home_exists
```

Comment:
I don't see this registered variable used anywhere - should we do something with it?

Reply:
Get rid of it!